playground/frontend/playground_components/tools/generate_build_metadata.sh

```diff
@@ -0,0 +1,39 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@alexeyinkin also, please run `shellcheck` linter tool on this script

Reply:
`+`